CRM/Member/BAO/MembershipType.php

```diff
@@ -620,15 +620,14 @@ public static function getRenewalDatesForMembershipType($membershipId, $changeTo
    *   array of the details of membership types
    */
   public static function getMembershipTypesByOrg($orgID) {
-    $membershipTypes = array();
-    $dao = new CRM_Member_DAO_MembershipType();
-    $dao->member_of_contact_id = $orgID;
-    $dao->find();
-    while ($dao->fetch()) {
-      $membershipTypes[$dao->id] = array();
-      CRM_Core_DAO::storeValues($dao, $membershipTypes[$dao->id]);
-    }
-    return $membershipTypes;
+    Civi::log()->warning('Deprecated function getMembershipTypesByOrg, please user membership_type api', array('civi.tag' => 'deprecated'));
+    $memberTypesSameParentOrg = civicrm_api3('MembershipType', 'get', array(
```

Comment:
I think some copy & pasting did go wrong here. I think you wanted to paste this into: CRM_Member_Page_Tab::browse() and leave the to be deprecated function as it is.

Reply:
I think this is fine - it does the same thing as the previous function but in a way that is more clearly 'the same as the api'